### ISSUE TITLE
specify books

### ISSUE THREAD

**Issue by jankoweb** [khornberg/bible-reading-planner#5]
Hi, would it be possible to add an option to specify book? (I want to create a reading book for some specific book that I need to read)

**Comment by khornberg** [comment]
So instead of a preset "sequence" you want the "sequence" to be a single bible book?

**Comment by jankoweb** [comment]
Yes. I mean something like http://www.studylight.org/plan/custom/index.cgi - to be able to select books, that I want to read now. 

Your application is unique that is able to set number of verses per a day:)

**Comment by khornberg** [comment]
I'll try to take a look soon. Were you thinking that you'd like multiple books to be selected or just one?

**Comment by jankoweb** [comment]
I will be pleasured when it can be more books:) (I have some books that I want to read in my own order)

**Comment by khornberg** [comment]
Thank you for the request and suggestion. I'll leave this issue open as a to do. I do not know when or if I'll get to adding the feature. I do not have much time any more.
